### PR TITLE
add filter on date for samples

### PR DESCRIPTION
**Pull request by robinandeer**
filter before transforming the entry to optimize for speed!

### CODE REVIEW

**Comment by robinandeer** [comment]
ping @mayabrandi @b4ckm4n

**Comment by ingkebil** [comment]
ping @b4ckm4n

File: lims/api.py [hunk 1]
```diff
@@ -11,6 +12,12 @@
 logger = logging.getLogger(__name__)
 
 
+def dateify(str_date, template="%Y-%m-%d"):
+    """Convert string to date object."""
```
Comment (b4ckm4n) [inline lims/api.py]: there's no explanation for the parameters

Comment (robinandeer) [inline lims/api.py]: good call!

**Pull request merged**